.github/workflows/sv_pipeline_docker.yml

```diff
@@ -203,24 +206,45 @@ jobs:
           sudo mv "$tmp" /etc/docker/daemon.json
           sudo systemctl restart docker.service
 
-      - name: Build and Publish Docker Images
+      - name: Build and Publish Docker Images to GCR
         id: build_and_publish
         run: |
           python ./scripts/docker/build_docker.py \
             --base-git-commit ${{ needs.build_args_job.outputs.base_sha }} \
             --current-git-commit ${{ needs.build_args_job.outputs.head_sha }} \
-            --gcr-project ${{ secrets.GCP_PROJECT_ID }}/gatk-sv \
+            --docker-repo us.gcr.io/${{ secrets.GCP_PROJECT_ID }}/gatk-sv \
```

Comment:
This would need to be changed to point to the Australian Artifact Registry, right?